.eslintrc

```diff
@@ -28,8 +36,13 @@
       "exports": "ignore",
       "functions": "never"
     }],
+    "no-use-before-define": 0,
+    "import/no-named-as-default": 0,
     "import/prefer-default-export": 1,
-    "max-len": ["error", {"code": 120}],
+    "max-len": 0,
+    "class-methods-use-this": 0,
```

Comment:
I'd like to add this back in the future, if we dont use `this`, its probably better off as a stateless react component (as just a function)